citadel/indico_citadel/search.py

```diff
@@ -53,27 +53,33 @@ def search(self, query, access, page=1, object_types=(), **params):
         return CitadelResultSchema(context={'results_per_page': self.RESULTS_PER_PAGE}).load(data)
 
     def get_placeholders(self):
-        return [SearchPlaceholder(key, label) for key, (_, label) in placeholders.items()]
+        return [SearchFilter(key, label) for key, (_, label) in placeholders.items()]
+
+    def get_filters(self):
+        return [SearchFilter(key, label) for key, label in filters.items()]
 
 
 placeholders = {
     'title': ('_data.title', _('An entry title (such as: event, contribution, attachment, event_note)')),
     'person': ('_data.persons.name', _("A speaker, author or event chair's name")),
     'affiliation': ('_data.persons.affiliation', _("A speaker, author or event chair's affiliation")),
     'type': ('_data.type', _('An entry type (such as: conference, meeting, link, file)')),
-    'venue': ('_data.location.venue_name', _("The event's venue name")),  # TODO: multiple locations
+    'venue': ('_data.location.venue_name', _("The entry's venue name")),
```

Comment:
What about rewording these titles like this?

> Name of the venue
Name of the room
Address of the venue
Name of the attached file

Especially the "entry" sounds a bit strange for end users

Reply:
Sounds good. I've renamed all of the listed above.